package.json

```diff
@@ -45,6 +45,7 @@
     "grunt-git": "0.3.7",
     "grunt-jscs": "2.6.0",
     "grunt-saucelabs": "8.6.2",
+    "grunt-tslint": "^4.0.1",
```

Comment:
We already `tslint` the src using npm scripts instead of using grunt (we'd like to move away from grunt in general; see #3210). I think you can just add a script to lint test files as well